OrbitCore/Path.cpp

```diff
@@ -65,9 +65,7 @@ std::string Path::GetExecutablePath() {
 }
 
 bool Path::FileExists(const std::string& file) {
-  struct stat statbuf;
-  int ret = stat(file.c_str(), &statbuf);
-  return ret == 0;
```

Comment:
I have not idea why this doesn't work on windows.  I agree that we should use filesystem::exists but still I am puzzled by this. Is this some kind of known bug in windows stat implementation?

Reply:
I tried to figure it out but i did not make any progress and decided its not worth the effort. 
It did not work for a module that had 3 dashes in the filename.